CHANGELOG.asciidoc

```diff
@@ -59,7 +59,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Made the url field of heartbeat aggregatable. {pull}6263[6263]
 
 *Metricbeat*
-
+- Add DNS Policy to system metricbeat. {pull}6656[6656] 
```

Comment:
Could you specify in the changelog that it refers to Kubernetes deployment files? For the rest it LGTM

Reply:
Done